### ISSUE TITLE
🐞 - [NGX-UI-TOUR-MD-MENU] close tour does not work with RouterModule.forRoot

### ISSUE THREAD

**Issue by ikrasnikov** [hakimio/ngx-ui-tour#173]
### Which ngx-ui-tour-* package is the source of the bug?

md-menu

### Please provide a link to a minimal reproduction of the bug

https://stackblitz.com/edit/angular-wau2z8-3lup83

### Description

If I trigger start tour in OnInit hook and use RouterModule.forRoot() 'Close' and 'End' tour buttons do not work.
Had to start tour in setTimeout(), or remove routs.

### Angular version

17

### Ngx UI Tour version

12

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [ ] macOS
- [ ] Windows
- [X] Linux
- [ ] iOS
- [ ] Android

**Comment by hakimio** [comment]
As a workaround until this is fixed you can add the following line before calling `tourService.initialize()`:
```ts
(this.tourService as any)['subscribeToNavigationStartEvent'] = () => {};
```

**Comment by ikrasnikov** [comment]
Works for me, thank you!

**Comment by hakimio** [comment]
Fixed in `v12.0.3`.
